projects/sourceware.org/bzip2/Makefile-libbz2_dylib

```diff
@@ -0,0 +1,44 @@
+# This Makefile builds a shared version of the library,
+# libbz2.dylib for MacOSX x86 (10.13.4 or higher),
+# It is a custom Makefile. Use at own risk.
+# Run in your MacOS terminal with the following command:
+# make -f Makefile-libbz2_dylib
+
+PKG_VERSION?=1.0.8
```

Comment:
Like, we should probably export `VERSION` like we do for virtual-envs for consistency.

Reply:
Yeah, I tried to make as few modifications to the third party makefile as possible, but that seems worthwhile.